app/src/main/res/layout/activity_main.xml

```diff
@@ -7,12 +7,91 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
     <TextView
+      android:id="@+id/city_name"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:text="Hello World!"
-      app:layout_constraintBottom_toBottomOf="parent"
+      android:text="@string/st_paul_mn"
+      android:layout_marginTop="12dp"
+        android:textSize="18dp"
```

Comment:
Make sure your code is properly aligned. You can run the code formatter in AndroidStudio